ui/tests/__tests__/FacetCard.test.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import FacetCard from "../../src/components/facets/FacetCard";
+import { getFacet } from "../mockutils";
+import { Checkbox, MuiThemeProvider } from "material-ui";
+import { getMuiTheme } from "material-ui/styles";
+import PropTypes from "prop-types";
+
+test("Renders correctly with input", () => {
+  const tree = shallow(
+    <FacetCard facet={getFacet("Facet Name", 3)} updateFacet={() => {}} />
+  );
+
+  expect(tree).toMatchSnapshot();
+  expect(tree.find(".listItem").length).toBe(3);
+  expect(tree.find(".totalFacetValueCount").text()).toBe("60");
+});
+
+test("Updates values after selecting checkboxes", () => {
```

Comment:
This test and below - There will be an integration test for this, I prefer not to have duplicate unit tests

Reply:
Since we don't currently have any integration/functional tests or know what those will look like and how much crossover with these tests they'll have, I'd opt to keep these tests to exercise key code for now.

Reply:
TODO to delete after functional tests are added?